### ISSUE TITLE
Set base paths on sonata-admin-integration-bundle

### ISSUE THREAD

**Issue by ElectricMaxxx** [symfony-cmf/core-bundle#217]
For an better configuration usage it would be nice to have the preprocessing that happend on each bundle before.

blocked by https://github.com/symfony-cmf/sonata-admin-integration-bundle/issues/22

**Comment by wouterj** [comment]
👎 The SonataAdminIntegrationBundle just uses the basepath parameters set by the core bundle. However, some bundles require special admin basepaths (like Routing). We never preprocessed these paths, so I don't think we should.

**Comment by ElectricMaxxx** [comment]
i just wanted to keep those paths that had been set by sonata before, or did sonata set the `use_sonata_admin` flag only?

**Comment by wouterj** [comment]
Yes

**Comment by ElectricMaxxx** [comment]
Oh. Then the pre-processing of the paths was in my head only? So this issue should stay there too :-)
